src/site/pages/home/Home.module.css

```
@@ -0,0 +1,174 @@
+	}
+
+	60% {
+		opacity: 1;
+		transform: perspective(400px) rotateX(10deg);
+	}
+
+	80% {
+		transform: perspective(400px) rotateX(-5deg);
+	}
+
+	to {
```

Stylelint (17.14.0)

[error] Expected "to" to be "100%" (keyframe-selector-notation)

(keyframe-selector-notation)